physics.py

```diff
@@ -4,50 +4,29 @@
 import typing
 import phoenix6
 import phoenix6.unmanaged
-import phoenix5
 import wpilib
 
 from pyfrc.physics.core import PhysicsInterface
 from wpimath.kinematics import SwerveDrive4Kinematics
 from wpilib.simulation import SimDeviceSim
 
 from components.chassis import SwerveModule
-from utilities.ctre import VERSA_ENCODER_CPR
 
 if typing.TYPE_CHECKING:
     from robot import MyRobot
 
 
 class SimpleTalonFXMotorSim:
-    def __init__(
-        self, motor: phoenix6.hardware.TalonFX, kV: float, rev_per_unit: float
-    ) -> None:
+    def __init__(self, motor: phoenix6.hardware.TalonFX, kV: float) -> None:
         self.sim_state = motor.sim_state
+        self.sim_state.set_supply_voltage(12.0)
```

Comment:
holy crap I can't believe it was _this_ simple, how did I miss this

Reply:
Yeah, I thought you'd hate this!